app/ui/client/views/app/room.js

```diff
@@ -254,6 +254,17 @@ callbacks.add('enter-room', wipeFailedUploads);
 const ignoreReplies = getConfig('ignoreReplies') === 'true';
 
 Template.room.helpers({
+	dragAndDrop() {
+		return settings.get('FileUpload_Enabled') && 'dropzone--disabled';
+	},
+
+	isDisabled() {
```

Comment:
We are already checking the `settings.get('FileUpload_Enabled')` on `dragAndDrop`. I believe with a bit of wording tweaks on the classes, we can use only one helper

Reply:
Done